stuffer/s2n_stuffer_pem.c

```diff
@@ -114,7 +115,15 @@ int s2n_stuffer_private_key_from_pem(struct s2n_stuffer *pem, struct s2n_stuffer
     
     s2n_stuffer_reread(pem);
     s2n_stuffer_reread(asn1);
-    rc = s2n_stuffer_data_from_pem(pem, asn1, S2N_PEM_PKCS1_ECDSA_PRIVATE_KEY);
+
+    /* Try EC private key, skipping EC PARAMETERS if it exists. For now, only support standard curves. */
```

Comment:
I think this comment needs to be expanded slightly. Your PR comments go into the details, but we should put some of those into the code as well. How does the following sound?

> By default, OpenSSL always outputs both "EC PARAMETERS" and "EC PRIVATE KEY" PEM Objects. Skip the first "EC PARAMETERS" Object so that we're compatible with OpenSSL's default output, and since "EC PARAMETERS" is only needed for non-standard curves that aren't currently supported.